README.md

```diff
@@ -267,14 +268,15 @@ semantics onto C++ such as Rust-inspired
 
 ## Getting started
 
-To try out Carbon, you can use the Carbon explorer to interpret Carbon code and
-print its output. You can try it out immediately at
+To try out Carbon immediately in your browser, you can use the Carbon explorer
+to interpret Carbon code and print its output on:
```

Comment:
You might consider putting a little more emphasis on "demo interpreter for Carbon", like above, rather than just "Carbon".

Reply:
Done I think, PTAL.